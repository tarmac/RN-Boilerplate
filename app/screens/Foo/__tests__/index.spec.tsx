import { shallow } from 'enzyme';
import React from 'react';
import FooScreen from '../index';

import { View, Text } from 'react-native';

describe('Test Foo Screen: ', () => {
  const wrapper = shallow(<FooScreen />);

  it('should match to snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should render one View component', () => {
    expect(wrapper.find(View)).toHaveLength(1);
  });

  it('should render 6 Text components', () => {
    expect(wrapper.find(Text)).toHaveLength(6);
  });
});
