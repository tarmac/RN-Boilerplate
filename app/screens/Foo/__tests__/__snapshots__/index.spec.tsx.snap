// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Foo Screen:  should match to snapshot 1`] = `ShallowWrapper {}`;
